lib/trento/settings/analytics_settings.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Trento.Settings.AnalyticsSettings do
+  @moduledoc """
+  Settings related to analytics.
+  """
+
+  use Ecto.Schema
+  use Trento.Support.Ecto.STI, sti_identifier: :analytics_settings
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{}
+
+  @derive {Jason.Encoder, except: [:__meta__, :__struct__]}
+  @primary_key {:id, :binary_id, autogenerate: true}
+  schema "settings" do
+    field :opt_in, :boolean, source: :analytics_settings_opt_in, default: false
+    sti_fields()
```

Comment:
I think we can add the `timestamps(type: :utc_datetime_usec)` macro in top of `sti_fields` so we add the `created_at/updated_at` fields. It is a good to have thing.
Check other settings schemas for reference